usr/src/uts/common/fs/zfs/arc.c

```diff
@@ -3424,8 +3424,8 @@ arc_reclaim_thread(void)
 			 * even if we aren't being signalled)
 			 */
 			CALLB_CPR_SAFE_BEGIN(&cpr);
-			(void) cv_timedwait(&arc_reclaim_thread_cv,
-			    &arc_reclaim_lock, ddi_get_lbolt() + hz);
+			(void) cv_timedwait_hires(&arc_reclaim_thread_cv,
+			    &arc_reclaim_lock, SEC2NSEC(1), MSEC2NSEC(1), 0);
```

Comment:
We have not noticed any difference and have been running this for the past 9 months. To minimize merge conflict with the compressed ARC changes that are coming next, I needed to integrate this change first.